examples/cat.cfg_template

```diff
@@ -1,10 +1,20 @@
 [kai_cat]
 topic=/EPS/0/
+; Command used for concatenation.
```

Comment:
For consistency, maybe use only `#` as comment character.

Reply:
Thanks — I didn't know that was valid syntax.

Reply:
Changed to `#`.